syntax = "proto3";

package server;

service IpDatabaseService {
    rpc getIp(IpRequest) returns (DatabaseResponse) {};
}

message IpRequest {
    string name = 1;
} 

message DatabaseResponse {
    string ip = 1;
}